### PR TITLE
Joining tables exercise

### PR DESCRIPTION
**Pull request by tinatsina**
## Main Changes
- added Wednesday challenge queries
- added screenshots

## Screenshots 📸 
image
image
image
image
image
image
image
image
image
image

## Review Changes 🛠️ 
image
image

### CODE REVIEW

**Review by Bantealem — CHANGES_REQUESTED** [review]
# Status: Change Required
Hi @tinatsina, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- [x] Correct Gti flow
- [x] Professional Readme
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: data.sql [hunk 1]
```diff
@@ -16,4 +16,43 @@ insert into animals(id,name,date_of_birth,escape_attempts,neutered,weight_kg) va
...
+/************ DAY 3 TASKS ************/
+/*************************************/
+
+insert into owners (full_name,age) values ('Sam Smith',34);
+insert into owners (full_name,age) values ('Jeniffer Orwell',19);
+insert into owners (full_name,age) values ('Bob',45);
+insert into owners (full_name,age) values ('Melody Pond',77);
+insert into owners (full_name,age) values ('Dean Winchester',14);
+insert into owners (full_name,age) values ('Jodie Whittaker',38);
+
```
[the remark this answers is not in the document]
Comment (tinatsina) [reply data.sql:31]: # Resolved
- took your advice and included the missing query for adding data into the species table.

File: schema.sql [hunk 2]
```diff
@@ -13,3 +13,31 @@ weight_kg real not null);
...
+alter table animals
+add column species_id int references species(id)
+on delete cascade;
+select * from animals;
+
+alter table animals
+add column owner_id int references owners(id)
+on delete cascade;
+select * from animals;
+
```
[the remark this answers is not in the document]
Comment (tinatsina) [reply schema.sql:43]: ## Resolved
- just to clarify the queries where added for me to verifiy as I am working on a terminal
- I needed to check if my queries had worked properly or not

File: schema.sql [hunk 3]
```diff
@@ -13,3 +13,31 @@ weight_kg real not null);
...
+create table species (
+	id SERIAL,
+	name varchar(32),
+	PRIMARY KEY (id)
+);
+
+alter table animals
+add column species_id int references species(id)
+on delete cascade;
+select * from animals;
```
[the remark this answers is not in the document]
Comment (tinatsina) [reply schema.sql:37]: ## Resolved
- I took your instructions and resolved this issue as well, the changes are done

File: schema.sql [hunk 4]
```diff
@@ -13,3 +13,31 @@ weight_kg real not null);
...
+  full_name varchar(32),
+  age int,
+  PRIMARY KEY (id)
+);
+
+create table species (
+	id SERIAL,
+	name varchar(32),
+	PRIMARY KEY (id)
+);
```
[the remark this answers is not in the document]
Comment (tinatsina) [reply schema.sql:32]: ## Resolved
- added new name length
- its 255 characters long

File: schema.sql [hunk 5]
```diff
@@ -13,3 +13,27 @@ weight_kg real not null);
...
+	id SERIAL,
+	name varchar(255),
+	PRIMARY KEY (id)
+);
+
+alter table animals
+add column species_id int references species(id);
+
+alter table animals
+add column owner_id int references owners(id);
```
Comment (Deepakdanger) [inline schema.sql:38]: - [ ] As requested by previous reviewer, Kindly drop the species column from the animal table before adding species_id & owner_id.
`ALTER TABLE animals DROP column species; `

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @tinatsina , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema.sql [hunk 6]
```diff
@@ -13,3 +13,27 @@ weight_kg real not null);
...
+  PRIMARY KEY (id)
+);
+
+create table species (
+	id SERIAL,
+	name varchar(255),
+	PRIMARY KEY (id)
+);
+
+alter table animals
```
Comment (tinatsina) [inline schema.sql:34]: # Resolved
- as requested I have dropped the species column before adding the `species_id` and `owners_id`

File: schema.sql (hunk 5 above)
Comment (tinatsina) [reply schema.sql:38]: ## Resolved
- i have made the adjustmernts
image

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @tinatsina 

Your project is complete! Great job for making the requested changes🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request
- screenshots well used

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**